website/client-old/js/controllers/notificationCtrl.js

```diff
@@ -32,6 +38,7 @@ habitrpg.controller('NotificationCtrl',
 
       if (yesterDailies.length === 0) {
         User.runCron();
```

Comment:
in a .then() maybe ? so it wait for the runCron request to finish (note that I've not checked if runCron returns a promise or not)